### PR TITLE
added steps bar

### LINKED ISSUE #250: add steps bar for js
**Describe the project you want to add with tech stack**
simple stepsbar for progress

**Expected behavior**
next and previous button to steps up and down

**Screenshots (optional)**

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by HridoyHazard**
# Title and Issue number 
Title : steps bar 

Issue No. : #250

Code Stack : Javascript [native]

Close #250

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/steps-bar/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="../../helpers/navbar.js" type="module"></script>
```
Comment (sadanandpai) [inline native/machine-coding/steps-bar/index.html:7]: header to be imported. Also, meta tags are no more needed

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by sadanandpai** [comment]
can we provide the field for the user to change the number of steps?

File: native/machine-coding/steps-bar/index.html (hunk 1 above)
Comment (HridoyHazard) [reply native/machine-coding/steps-bar/index.html:6]: hi i fixed the issue but i don't get you change number of steps meaning ? actually it basically progress steps bar. each next click will progress the bar and goes like this.

**Comment by HridoyHazard** [comment]
> can we provide the field for the user to change the number of steps?

hi i fixed the issue. but it's basically steps progress bar so when the user click it will progress to steps.

**Pull request closed without merging**